layouts/partials/article/components/details.html

```diff
@@ -23,7 +23,7 @@ <h3 class="article-subtitle">
         {{ end }}
     </div>
 
-    {{ if or (not .Date.IsZero) (.Site.Params.article.readingTime) }}
+    {{ if or (not .Date.IsZero) (.Params.article.readingTime) }}
```

Comment:
`.Params.article.readingTime` would correspond to the frontmatter field `article.readingTime`, which is not the correct one (`readingTime` only).

It's better to extract this part of the conditional since it's repeated in line 37 again.

I'm sorry, but I'm going to close this PR and do this part myself 🙏.